Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -249,11 +256,10 @@ open class NavigationMapView: UIView {
                     initPrimaryRoutePoints(route: route)
                     setUpLineGradientStops(along: route)
                 }
-            }
-            
-            if showsRestrictedAreasOnRoute {
-                parentLayerIdentifier = addRouteRestrictedAreaLayer(route,
-                                                                    below: parentLayerIdentifier)
+                
+                if showsRestrictedAreasOnRoute {
+                    parentLayerIdentifier = addRouteRestrictedAreaLayer(route, below: parentLayerIdentifier)
```

Comment:
Why is it for the first route only now?

Reply:
Showing restricted ares only on main route now since it did not work correctly for alternative routes. The same restricted area source was used for all routes. Spreading this feature to all visible routes would require significant change in this logic which I consider outside of the scope of this PR.